honeybadger.go

```diff
@@ -24,8 +47,37 @@ func (hb *HoneyBadger) sendBatch() {
 }
 
 // Create batch polling random transaction in queue
-func (hb *HoneyBadger) createBatch() {
-	panic("implement me w/ test case :-)")
+func (hb *HoneyBadger) createBatch() []Transaction {
+	minVal := min(hb.b, hb.que.Len())
+	buf := make([]Transaction, minVal)
+	pol := make([]Transaction, int(hb.b/hb.n))
```

Comment:
`int(hb.b/hb.n)` is reused later, so I suggest making a new variable and reuse that variable!